packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -246,34 +245,23 @@ async function redeemSharingLink(
 				});
 			}
 
-			const disableUsingTenantDomain = loggerToMonitoringContext(logger).config.getBoolean(
-				"Fluid.Driver.Odsp.DisableUsingTenantDomainForSharesApi",
-			);
+			const details = JSON.stringify({
```

Comment:
nit (optional): You don't need `JSON.stringify` here, the logger will do that for you before sending.